src/test/java/com/jcabi/ssl/maven/plugin/CacertsTest.java

```diff
@@ -56,17 +58,27 @@
      */
     @Test
     public void importsCertificatesFromKeystore() throws Exception {
-        final File keystore = this.temp.newFile("keystore.jks");
-        keystore.delete();
-        final File truststore = this.temp.newFile("cacerts.jks");
-        truststore.delete();
+        final File keystoreFile = this.temp.newFile("keystore.jks");
+        keystoreFile.delete();
+        final File truststoreFile = this.temp.newFile("cacerts.jks");
+        truststoreFile.delete();
         final String pwd = "some-password";
-        new Keystore(pwd).activate(keystore);
-        new Cacerts(truststore).imprt();
+        new Keystore(pwd).activate(keystoreFile);
```

Comment:
@yegor256 I've read the article and see that it's better to have as short variable name as possible.
And composite name can be a sign that the variable name is too long.

But in the method, we're talking about, we have vars for a file of  truststore and the truststore itself. So, I don't see I can simplify the truststoreFile name.
And since I use truststoreFile for a file of  truststore, then I should use keystoreFile for a file of  keystore.

I'd be pleased to know your opinion on this.